x-pack/plugins/lists/server/services/exception_lists/build_exception_filter.test.ts

```diff
@@ -479,6 +485,113 @@ describe('build_exceptions_filter', () => {
         }
       `);
     });
+
+    test('it should remove all exception items that are expired', async () => {
+      const futureDate = new Date(Date.now() + 1000000).toISOString();
+      const expiredDate = new Date(Date.now() - 1000000).toISOString();
+      const exceptions = [
+        { ...getExceptionListItemSchemaMock(), expire_time: futureDate },
```

Comment:
Using the same exception here 3 times makes it impossible to tell if the correct exceptions were included in the final filter.